presto-orc/src/main/java/io/prestosql/orc/OrcWriter.java

```diff
@@ -471,10 +477,11 @@ public void close()
                 .mapToLong(stripe -> stripe.getStripeInformation().getNumberOfRows())
                 .sum();
 
-        Optional<ColumnMetadata<ColumnStatistics>> fileStats = toFileStats(closedStripes.stream()
+        fileStats = toFileStats(closedStripes.stream()
                 .map(ClosedStripe::getStatistics)
                 .map(StripeStatistics::getColumnStatistics)
                 .collect(toList()));
+        fileStats.ifPresent(stats -> fileStatsRetainedBytes = stats.stream().mapToLong(ColumnStatistics::getRetainedSizeInBytes).sum());
```

Comment:
Assigning a field inside a lambda is not typical and easy to miss, so it'd be better to write like
```java
fileStatsRetainedBytes = fileStats.map(stats -> stats.stream()
        .mapToLong(ColumnStatistics::getRetainedSizeInBytes)
        .sum()).orElse(0L);